lib/utils/oclif-utils.ts

```diff
@@ -54,6 +54,5 @@ export function capitanoizeOclifUsage(
 ): string {
 	return (oclifUsage || '')
 		.toString()
-		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, match => `<${match}>`)
-		.toLowerCase();
+		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, match => `<${match}>`);
 }
```

Comment:
These changes required to preserve camelCase in usages.

Reply:
(The regex replace on ln57 doesn't seem to have any effect as far as I can tell).

Reply:
The regex was meant to add the angle brackets in Capitano style for the `balena help` output only -- so the angle brackets are not added to `cli.markdown`.